index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Personal Portfolio</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+</head>
+
+<body>
+
+  <header>
+    <div class="wrapper menu">
+      <span class="menu-logo">My Logo</span>
+      <span class="menu-button"></span>
```

Comment:
- [ ] Wow! You implemented the hamburger icon for mobile navigation.:grinning: However, I noticed that you have used a `<span>` tag to wrap the icon. It is recommended to wrap the icon within an `<a>` tag for two main reasons: semantics and accessibility. Additionally, the `<a>` tag should be enclosed within a `<nav>` tag to provide proper semantic structure to your navigation section.

:pray: Please make the necessary changes by wrapping the hamburger icon within an `<a>` tag and enclosing it within a `<nav>` tag to enhance the semantic structure and accessibility of your mobile navigation. To learn more [check this](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)